Exercise/compute_lowest_terms.py

```diff
@@ -1,31 +1,29 @@
 ## TODO: complete the function "lowest_terms" below
 #x = str(input('pls input the fraction: '))
 def lowest_terms(x):
-	# Testing denominator value equals zero
+	#converting input x into  wo integer
```

Comment:
your function should have a docstring. See slack discussion for help on writing docstrings